layers/+web-services/search-engine/packages.el

```diff
@@ -38,7 +38,8 @@
       (setq search-engine-alist
             `((amazon
                :name "Amazon"
-               :url "https://www.amazon.com/s/ref=nb_sb_noss?url=search-alias%%3Daps&field-keywords=%s")
+               :url (concat (format "https://www.amazon.%s" search-engine-amazon-tld)
```

Comment:
You can combine concat and format into one format form.

Reply:
Thanks for the prompt review, @lebensterben.
I've had some problems with that because of insufficient arguments for format. I guess that last `%s` must be escaped some way. Disclaimer: Elisp newbie here :sweat_smile:
My only concern, if I should go for that, is the lost readability, which made me do it this way in the end. Any advice?

Reply:
Well, you can (concat) everything.

Reply:
Done. I can't remember exactly which, but I was getting some type error when doing it this way in the beginning, so I worked around it but using `format`. Still finding my way around Elisp... 😅